llitgi/DataProvider/CoreData/CoreDataFactory.swift

```diff
@@ -81,25 +82,14 @@ final class CoreDataFactoryImplementation: CoreDataFactory {
         switch type {
         case .all:
             typePredicate = NSPredicate(format: "status_ != '2'")
-            let addedTime = NSSortDescriptor(key: "timeAdded_", ascending: false)
-            let id = NSSortDescriptor(key: "id_", ascending: false)
-            request.sortDescriptors = [addedTime, id]
         case .myList:
             typePredicate = NSPredicate(format: "status_ == '0'")
-            let addedTime = NSSortDescriptor(key: "timeAdded_", ascending: false)
-            let id = NSSortDescriptor(key: "id_", ascending: false)
-            request.sortDescriptors = [addedTime, id]
         case .favorites:
             typePredicate = NSPredicate(format: "isFavorite_ == true")
-            let timeUpdated = NSSortDescriptor(key: "timeUpdated_", ascending: false)
```

Comment:
This change makes me cringe, but it solves the assert that triggers on the FRC. I need to spend more time investigating this, because I'm pretty sure that the issue happens because we override the `timeUpdated_` property and that one is also changed when the `sync` operation happens.